src/go/k8s/apis/redpanda/v1alpha1/cluster_types.go

```diff
@@ -185,16 +185,16 @@ type RestartConfig struct {
 type PDBConfig struct {
 	// Enabled specifies whether PDB should be generated for the cluster. It defaults to true
 	Enabled bool `json:"enabled,omitempty"`
-	// An eviction is allowed if at least "minAvailable" pods selected by
+	// An eviction is allowed if at least "minAvailable" Pods selected by
 	// "selector" will still be available after the eviction, i.e. even in the
-	// absence of the evicted pod.  So for example you can prevent all voluntary
+	// absence of the evicted Pod.  So for example you can prevent all voluntary
 	// evictions by specifying "100%". This is a mutually exclusive setting with "maxUnavailable".
 	// you can read more in https://kubernetes.io/docs/tasks/run-application/configure-pdb/
 	// +optional
 	MinAvailable *intstr.IntOrString `json:"minAvailable,omitempty"`
-	// An eviction is allowed if at most "maxUnavailable" pods selected by
+	// An eviction is allowed if at most "maxUnavailable" Pods selected by
 	// "selector" are unavailable after the eviction, i.e. even in absence of
-	// the evicted pod. For example, one can prevent all voluntary evictions
+	// the evicted Pod. For example, one can prevent all voluntary evictions
```

Comment:
nit: just curious why is Pod capital? :)

Reply:
k8s resources are proper nouns. Making this distinction helps comprehension, especially with Node and Service. I forget why this caught my attention but I did put it in its own commit in case it was hated 😁